src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+import React from "react"
+import Helmet from "react-helmet"
+import {
+    Navbar, 
+    Footer
+} from "../components"
+import s from "styled-components"
+import { Title, Text, Subtitle, Link } from "../components/shared/Typography"
+import Button from '../components/shared/Button'
+import {GREEN} from "../utils/constants.js"
+import Dennis from '../images/dennis woodside.jpg'
+import Lidiya from '../images/lidiya-dervisheva.jpeg'
+import Seth from '../images/seth.jpg'
+import Marissa from '../images/marisa sweeney.jpg'
+import Sam from '../images/sam bursten.jpeg'
+
+const Headshot = s.img`
+    width: 13vw;
+    height: 13vw;
+    object-fit: cover;
+    border-radius: 50%;
+    margin: 1vw;
+`
+
+const Columns = s.div`
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    margin-top: 2vw;
+`
+const Caption = s.figcaption`
+    font-family: roboto;
+    font-weight: bold;
+    font-size: 20px;
+`
+
+const EventGroup = s.div`
+    margin: 2vw 0;
+`
+
+const EventNoDate = s.div`
+    border-left: 0.3rem solid ${GREEN};
+    padding: 0vw 1vw 0vw;
+    margin: 1vw 2vw;
+`
+
+const EventWithDate = s.div`
+    margin: 1vw 0;
+`
```

Comment:
Might be worthwhile to rename these as these are not Events, but rather Spacers/Frames.